dbt_platform_helper/utils/versioning.py

```diff
@@ -104,22 +104,40 @@ def get_github_released_version(repository: str, tags: bool = False) -> Tuple[in
     return parse_version(package_info["tag_name"])
 
 
+def get_latest_release():
```

Comment:
If this was extracted and has no tests for itself, it should probably be prefixed with "_" to indicate it's private.

Reply:
I have actually made this function more robust and added tests in another branch.  I was worried the scope of this PR was getting a bit large, but I could pull those changes in if you don't mind?

Reply:
Maybe it should be part of an independent piece of work because I'm not sure what we really want to return in the requests error cases at the moment! But feel free to take a look and add your thoughts.  I'm happy to move get_latest_release to a private function for now.

Reply:
Make it private for now, and address the error and how to handle it in the other ticket.